site/content/docs/5.3/examples/e-shop/index.html

```diff
@@ -0,0 +1,1567 @@
+            <a href="#" class="stretched-link">
+              <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false">
+                <use xlink:href="#chevron-right"/>
+              </svg>
+              <span class="visually-hidden">Go to xxxxxxx x xxx details</span>
+            </a>
+          </div>
+        </div></div>
+      </div>
+
+      <div class="col">
+        <div class="position-relative"><div class="row mt-md-3">
+          <div class="col-3"><div class="ratio ratio-3x4">{{< placeholder width="100%" height="100" text="icon" >}}</div></div>
+          <div class="col-7 col-md-5">
+            <h3 class="mb-1 fs-4 fw-bold">xxxxxxx xx xxxx</h3>
+            <p class="mb-0">
+              From<br>
+              <span class="fs-4 fw-bold text-primary">42 900 FCFA</span>
+            </p>
+          </div>
+          <div class="d-flex col-2 offset-md-1 justify-content-end">
+            <a href="#" class="stretched-link">
+              <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false">
+                <use xlink:href="#chevron-right"/>
+              </svg>
+              <span class="visually-hidden">Go to xxxxxxx xx xxxx details</span>
+            </a>
+          </div>
+        </div></div>
+      </div>
+
+      <div class="col">
+        <div class="position-relative"><div class="row mt-md-3">
+          <div class="col-3"><div class="ratio ratio-3x4">{{< placeholder width="100%" height="100" text="icon" >}}</div></div>
+          <div class="col-7 col-md-5">
+            <h3 class="mb-1 fs-4 fw-bold">xxxxx xxx xx</h3>
+            <p class="mb-0">
+              From<br>
+              <span class="fs-4 fw-bold text-primary">42 900 FCFA</span>
+            </p>
+          </div>
+          <div class="d-flex col-2 offset-md-1 justify-content-end">
+            <a href="#" class="stretched-link">
+              <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false">
+                <use xlink:href="#chevron-right"/>
+              </svg>
+              <span class="visually-hidden">Go to xxxxx xxx xx details</span>
+            </a>
+          </div>
+        </div></div>
+      </div>
+    </div>
+
+    <h2 class="visually-hidden" id="advices">Advices</h2>
+    <div class="row gy-3">
+      <div class="col-12 pt-3">
+        <div class="row gx-0 bg-dark">
+          <div class="col-12 col-lg-7 order-lg-1" aria-hidden="true">
+            <div class="ratio ratio-16x9">{{< placeholder width="100%" height="162" text="icon" >}}</div>
+          </div>
+          <div class="col-12 col-lg-5">
+            <div class="h-100 p-2 p-lg-4">
+              <div class="pt-1 pt-lg-0 pe-3 pe-lg-4 pb-1 ps-2">
+                <h3 class="h1 mb-2 text-primary">The best for my budget</h3>
+                <p class="h2 pt-1 pt-lg-2 mb-2">iPhone at a reduced price refurbished, with 1 year guarantly</p>
```

Comment:
`1 year guarantee` instead of `1 year guarantly`?

Reply:
TBD with designers